docs/conf/berlin/2025/news/announcing-cfp.rst

```diff
@@ -0,0 +1,67 @@
+:template: {{year}}/generic.html
+:banner: _static/conf/images/headers/portland-2024-speakers.jpg
+:og:image: _static/conf/images/headers/portland-2024-speakers.jpg
+
+.. post:: Jun 05, 2025
+   :tags: {{shortcode}}-{{year}}, website, cfp, tickets
+
+Announcing the Call for Proposals and Sponsorship Prospectus
```

Comment:
I'm a little confused, I thought we were announcing ticket sales and the website as well?

Reply:
I'll throw some more stuff in. I just mean this as a  starting point not as a finished post.